lib/features.js

```diff
@@ -23,7 +23,9 @@ var variables = require('rework-vars');
 
 exports.import = function(options){
   return 'undefined' == typeof window && options.source
-    ? importer()
+    ? importer({
+      path: options.paths
```

Comment:
I went to `paths` since `path` implies a single string when it can handle an array